cypress/integration/api_calls_spec.js

```diff
@@ -0,0 +1,26 @@
+describe('API Calls Error Messages', () => {
```

Comment:
I am SO PROUD that we did this app this way. It will be amazing to reference in the future.